fatiando/tests/mesher/test_mesher.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+from ...mesher import Square
+
+
+def test_square_copy():
+    orig = Square([0, 1, 2, 4], {'density': 750})
+    cp = orig.copy()
```

Comment:
Another check you do is to check is an object is the same reference as another using the `is` comparison: `cp is not orig`. 

Two objects might be equal but not be the same object.